src/java.compiler/share/classes/javax/lang/model/util/Types.java

```diff
@@ -34,6 +34,16 @@
 
 /**
  * Utility methods for operating on types.
+
+ * Most methods operate on {@linkplain PrimitiveType primitive types},
+ * {@linkplain ReferenceType reference types} (including {@linkplain
+ * ArrayType array types} and the {@linkplain NullType null type}),
+ * {@linkplain IntersectionType intersection types}, and the
+ * pseudo-type '{@link TypeKind#VOID void}'. {@linkplain
+ * ExecutableType Executable types} and the pseudo-types for
+ * {@linkplain TypeKind#PACKAGE packages} and {@linkplain
+ * TypeKind#MODULE modules} are generally out of scope for these
+ * methods.
```

Comment:
Do you want to better specify the behavior for the "out of scope" types, such as _Unless otherwise specified, `IllegalArgumenntException` will be thrown when a method is called with an inappropriate argument._

Reply:
Good suggestion; blanket statement added.